src/dom-updates.js

```diff
@@ -1,5 +1,56 @@
 const domUpdates = {
+  displayUserName(user) {
+    const nameDisplay = document.getElementById('welcome')
+    nameDisplay.innerText = `Welcome ${user.returnFirstName()}`
+  },
 
+  displayAmountSpent(user) {
+    const amountSpentDisplay = document.getElementById('amountSpent')
+    amountSpentDisplay.innerText = `You have spent $${user.calculateAmountSpent()} traveling`
+  },
+
+  displayUserTrips(user) {
+    let todaysDate = "2020/3/28"
+    const pastTripList = document.getElementById('pastTripList')
+    const upcomingTripList = document.getElementById('upcomingTripList')
+    const ongoingTripList = document.getElementById('ongoingTripList')
+    const pendingTripList = document.getElementById('pendingTripList')
+
+    user.trips.forEach((trip, i) => {
```

Comment:
Like the way you organized the trips and kept it really straightforward and organized